controllers/comments.controllers.js

```diff
@@ -23,4 +23,15 @@ exports.postComment = (req, res, next) => {
         res.status(201).send({ comment })
     })
     .catch(next)
+}
+
+exports.deleteComment = (req, res, next) => {
+    const { comment_id } = req.params;
+    const commentPromises = [checkCommentExists(comment_id), removeComment(comment_id)]
+
+    Promise.all(commentPromises)
```

Comment:
This could actually be done all in one by adding a 'returning *' to your psql query to see the deleted rows. If those rows are empty then you would have a 404 where that comment id doesn't exist. This is fine too, but does make two database queries.